src/octoprint/util/comm.py

```diff
@@ -2017,6 +2061,17 @@ def _handle_errors(self, line):
 		# finally return the line
 		return line
 
+	def _filterXonXoffCharacters(self, ret):
```

Comment:
Let's use `serial.XON` and `serial.XOFF` instead of redefining those constants.

Reply:
thank you, I didn't knew they where defined in serial..